.github/workflows/release-tag.yml

```
@@ -25,7 +25,11 @@ jobs:
       run: |
         echo "This workflow should not be triggered on main"
         exit 1
-    
+        
+    - name: Check version format
+      if: github.ref == 'refs/heads/main'
+      run: echo '${{ inputs.version }}' | grep -P '^\d+\.\d+\.\d+(\.[a-z0-9]+)?$'
```

Here the same applies:
```suggestion
    - name: Validate version format
      run: echo '${{ inputs.version }}' | grep -P '^\d+\.\d+\.\d+(\.[a-z0-9]+)?$'
```